inc/datahandlers/user.php

```diff
@@ -1536,17 +1536,24 @@ function delete_content($delete_uids=false)
 		$plugins->run_hooks('datahandler_user_delete_content', $this);
 
 		$db->delete_query('userfields', "ufid IN({$this->delete_uids})");
-		$db->delete_query('privatemessages', "uid IN({$this->delete_uids})");
+		$db->delete_query('privatemessages', "uid IN({$this->delete_uids}) OR fromid IN({$this->delete_uids})");
```

Comment:
Maybe. But where should it be moved then? `delete_posts()` would be the more proper alternative perhaps since it takes care of posts/threads but the name of the function might be confusing.